app/views/mailer/waitlist_email.html.erb

```diff
@@ -0,0 +1,6 @@
+<h3>Hold tight, <%= @questionnaire.first_name %>.</h3>
+<p>Unfortunately your application to BrickHack 4 has been waitlisted. But it's not over yet! We are still waiting for a final count and we will review your application at a later date.</p>
```

Comment:
"Will review" sort of conveys the applicant will definitely get a final approval/denial decision before the event. Not sure on exactly how the waitlist process will work, but if the applicant will sit in waitlist state *unless* they are accepted (aka no official denial notification), it might be better to say "We are still waiting for a final count and will let you know if space opens up."